catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/converter/DorisTypeConverter.java

```diff
@@ -10,15 +10,96 @@
 
 /** Type converter for Doris. */
 public class DorisTypeConverter extends JdbcTypeConverter<String> {
-  // TODO: add implementation for doris catalog
+  static final String BOOLEAN = "boolean";
+  static final String TINYINT = "tinyint";
+  static final String SMALLINT = "smallint";
+  static final String INT = "int";
+  static final String BIGINT = "bigint";
+  static final String FLOAT = "float";
+  static final String DOUBLE = "double";
+  static final String DECIMAL = "decimal";
+  static final String DATETIME = "datetime";
+  static final String CHAR = "char";
+  static final String STRING = "string";
 
   @Override
   public Type toGravitinoType(JdbcTypeBean typeBean) {
-    return Types.UnparsedType.of(typeBean.getTypeName());
+    switch (typeBean.getTypeName().toLowerCase()) {
+      case BOOLEAN:
+        return Types.BooleanType.get();
+      case TINYINT:
+        return Types.ByteType.get();
+      case SMALLINT:
+        return Types.ShortType.get();
+      case INT:
+        return Types.IntegerType.get();
+      case BIGINT:
+        return Types.LongType.get();
+      case FLOAT:
+        return Types.FloatType.get();
+      case DOUBLE:
+        return Types.DoubleType.get();
+      case DECIMAL:
+        return Types.DecimalType.of(
+            Integer.parseInt(typeBean.getColumnSize()), Integer.parseInt(typeBean.getScale()));
+      case DATE:
+        return Types.DateType.get();
+      case DATETIME:
+        return Types.TimestampType.withTimeZone();
+      case CHAR:
+        return Types.FixedCharType.of(Integer.parseInt(typeBean.getColumnSize()));
```

Comment:
Should we need to check the validation of the length of `char(n)`, `varchar(n)` for Doris.

Reply:
`toGravitinoType` will convert column type getted from Doris, so I think the length is certainly legitimate

Reply:
OK